technical-resources/security/security.md

```diff
@@ -19,6 +19,16 @@ Maple Finance V2 went through 2 audits during its development for the June 2023
 | Spearbit Auditors via Cantina | [`2023-06-05 - Cantina Report`](https://docs.google.com/viewer?url=https://github.com/maple-labs/maple-v2-audits/files/11667848/cantina-maple.pdf) |
 | Three Sigma | [`2023-04-10 - Three Sigma Report`](https://docs.google.com/viewer?url=https://github.com/maple-labs/maple-v2-audits/files/11663546/maple-v2-audit_three-sigma_2023.pdf) |
 
+<br>
+
+## December 2023 Release
+Maple Finance V2 went through 2 audits during its development for the December 2023 release, details of which you can find below. All relevant issues identified by auditors were addressed prior to release.
```

Comment:
Something that comes from previous versions, but I think we should edit the `Maple Finance V2 went` to not explicitly reference a v2. I think using simple `the contracts went through ....`